tests/Resources/wasm.test.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+/* eslint max-len: "off" */
+/* eslint no-undef: "off" */
+/* eslint comma-spacing: "off" */
+/* eslint array-bracket-spacing: "off" */
+'use strict';
+const should = require('./utilities/assertions');
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/w...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/wasm.test.js;
  Line: 14;
-->

- :warning: tests/Resources/wasm.test.js line 14 – 'should' is assigned a value but never used. (no-unused-vars)